src/freeze-dry/common.test.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-env jest */
+
+import { inlineUrlsInAttributes, urlToDataUri } from 'src/freeze-dry/common'
+import * as responseToDataUri from 'src/util/response-to-data-uri'
+
+describe('urlToDataUri', () => {
+    test('should return a valid dataUri for a URL', async () => {
+        const spy = jest.spyOn(responseToDataUri, 'default')
+        await urlToDataUri('https://example.com/page')
+        expect(spy).toHaveBeenCalled()
+        spy.mockRestore()
+    })
+
+    test('should return a "about:blank" for a invalid URL', async () => {
+        expect.assertions(2)
+        const spy = jest.spyOn(responseToDataUri, 'default').mockImplementation(() => {
+            throw new Error('mock error')
+        })
+        const dataUri = await urlToDataUri('example.com')
+        expect(spy).toHaveBeenCalled()
+        expect(dataUri).toBe('about:blank')
+        spy.mockRestore()
+    })
+
+    test('should return a "about:invalid" for an error URL' ,async () => {
+        fetch.mockResponseOnce(new Error())
+        const dataUri = await urlToDataUri('example.com')
+        expect(dataUri).toBe('about:blank')
+    })
+
+    test('should return the same url if already in dataUri form', async () => {
```

Comment:
According to @Treora review I added the two tests. I hope these were the test cases asked. But for some reason the tests seem to both fail. Any input would be helpful